packages/cli/__tests__/zosfiles/__unit__/create/ds/ds.handler.unit.test.ts

```diff
@@ -0,0 +1,102 @@
+            // Require the handler and create a new instance
+            const handlerReq = require("../../../../../src/zosfiles/create/ds/ds.handler");
+            const handler = new handlerReq.default();
+            const dataSetName = "testing";
+            const likeDataSetName = "testing";
+
+            // Vars populated by the mocked function
+            let error;
+            let apiMessage = "";
+            let jsonObj;
+            let logMessage = "";
+            let AbstractSession = null;
+
+            // Mock the create function
+            Create.dataSetLike = jest.fn((session) => {
+                AbstractSession = session;
+                return {
+                    success: true,
+                    commandResponse: "created"
+                };
+            });
+
+            // Mocked function references
+            const profFunc = jest.fn((args) => {
+                return {
+                    host: "fake",
+                    port: "fake",
+                    user: "fake",
+                    password: "fake",
+                    auth: "fake",
+                    rejectUnauthorized: "fake"
+                };
+            });
+
+            try {
+                // Invoke the handler with a full set of mocked arguments and response functions
+                await handler.process({
+                    arguments: {
+                        $0: "abstract",
+                        _: ["abstract"],
+                        dataSetName,
+                        likeDataSetName,
+                        ...UNIT_TEST_ZOSMF_PROF_OPTS
+                    },
+                    response: {
+                        data: {
+                            setMessage: jest.fn((setMsgArgs) => {
+                                apiMessage = setMsgArgs;
+                            }),
+                            setObj: jest.fn((setObjArgs) => {
+                                jsonObj = setObjArgs;
+                            })
+                        },
+                        console: {
+                            log: jest.fn((logArgs) => {
+                                logMessage += "\n" + logArgs;
+                            })
+                        },
+                        progress: {
+                            startBar: jest.fn((parms) => {
+                                // do nothing
+                            }),
+                            endBar: jest.fn(() => {
+                                // do nothing
+                            })
+                        }
+                    },
+                    profiles: {
+                        get: profFunc
+                    }
+                } as any);
+            } catch (e) {
+                error = e;
+            }
+
+            expect(error).toBeUndefined();
+            expect(profFunc).toHaveBeenCalledWith("zosmf", false);
+            expect(Create.dataSet).toHaveBeenCalledTimes(1);
```

Comment:
Please change `Create.dataSet` to `Create.dataSetLike` as done on the line below. `Create.dataSetLike` is the only SDK method being called by the "create ds" command and mocked for this test.